### ISSUE TITLE
No atomic parameters with python3 cause traceback

### ISSUE THREAD

**Issue by ybznek** [projectatomic/atomic#242]
python2
```
$ sudo atomic
/bin/atomic: too few arguments
Try 'atomic --help' for more information.
```

python3
```
$ sudo atomic
Traceback (most recent call last):
  File "/bin/atomic", line 399, in <module>
    sys.exit(args.func())
AttributeError: 'Namespace' object has no attribute 'func'
```

**Comment by ashcrow** [comment]
@ybznek can you provide a bit more information? I'm unable to reproduce this on:

- OS: Fedora 21
- Python: 3.4.2
- Version: 1.7
- Git Hash: 95b28a9ffd0f23a3a45a5e7cde40cb5341ef2f11
- Install Type: Source (python3 setup.py install)

```
$ sudo atomic
usage: atomic [-h] [-v]
              {diff,info,install,images,migrate,mount,push,upload,scan,stop,run,uninstall,unmount,update,version,verify}
              ...
```

**Comment by rhatdan** [comment]
This was a bug in f23 python, but newer atomic clients should fix it.

See if it fixed in atomic-1.6-5.git09ac479.fc23

**Comment by cgwalters** [comment]
FWIW this is waiting on https://bodhi.fedoraproject.org/updates/FEDORA-2015-825b7a03c1
